synphot/models.py

```diff
@@ -611,6 +618,16 @@ def sampleset(self, factor_step=0.1, **kwargs):
 
         return np.asarray(w)
 
+    # TODO: Remove unit hardcoding when we use model with units natively.
+    def integrate(self, x):
+        # TODO: Fix the formula, apply units, check for n_models>1
+        sig2 = self.sigma * self.sigma
+        dx_min = min(x) - self.x_0
+        dx_max = max(x) - self.x_0
+        a1 = dx_min * np.exp(-0.5 * dx_min * dx_min / sig2)
+        a2 = dx_max * np.exp(-0.5 * dx_max * dx_max / sig2)
+        return -1 * self.amplitude * (a2 - a1) / sig2
```

Comment:
To make things easier...
```suggestion
        return self.amplitude * (a2 - a1)
```